scripts/rabbitmq-env.bat

```diff
@@ -418,7 +417,7 @@ exit /b
 
 :filter_path
 REM Ensure ERL_LIBS begins with valid path
-IF [%ERL_LIBS%] EQU [] (

```

Comment:
According to [this document](https://ss64.com/nt/equ.html) `==` is meant to be used for string comparison anyway.